WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostListFragment.java

```diff
@@ -84,6 +84,10 @@
         implements ReaderInterfaces.OnPostSelectedListener,
                    ReaderInterfaces.OnTagSelectedListener,
                    ReaderInterfaces.OnPostPopupListener {
+    private static final String READER_DETAIL_TYPE_KEY = "post-detail-type";
+    private static final String READER_DETAIL_TYPE_NORMAL = "normal";
+    private static final String READER_DETAIL_TYPE_BLOG_PREVIEW = "preview-blog";
+    private static final String READER_DETAIL_TYPE_TAG_PREVIEW = "preview-tag";
```

Comment:
100% agree, but I couldn't find any existing bucket for these types of strings. Maybe a good spot would be around [here](https://github.com/wordpress-mobile/WordPress-Android/blob/develop/libs/analytics/WordPressAnalytics/src/main/java/org/wordpress/android/analytics/AnalyticsTracker.java#L14) or [here](https://github.com/wordpress-mobile/WordPress-Android/blob/develop/libs/analytics/WordPressAnalytics/src/main/java/org/wordpress/android/analytics/AnalyticsTrackerMixpanel.java#L33).